src/Chapter16/Listing16.12.SelectingWithoutTheQueryContinuation.cs

```diff
@@ -1,29 +1,34 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter16.Listing16_12
 {
     using AddisonWesley.Michaelis.EssentialCSharp.Chapter16.Listing16;
+    #region INCLUDE
     using System;
     using System.Collections.Generic;
     using System.Linq;
-
+    #region EXCLUDE
     public class Program
     {
         public static void Main()
         {
             GroupKeywords1();
         }
-
+        #endregion EXCLUDE
         private static void GroupKeywords1()
         {
-            IEnumerable<(bool IsContextualKeyword, IGrouping<bool, string> Items)> selection =
+            IEnumerable<
+                (bool IsContextualKeyword, IGrouping<bool, string> Items)> 
+                    selection =
```

Comment:
```suggestion
            IEnumerable<(bool IsContextualKeyword, 
                IGrouping<bool, string> Items)> selection =
```
I think this is a better split point